modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -1451,6 +1452,87 @@ public void testUpdateObjectAction() throws Exception {
 				"url", "https://onafterdelete.com"
 			).build(),
 			ObjectActionConstants.STATUS_NEVER_RAN);
+
+		String externalReferenceCode2 = RandomTestUtil.randomString();
+
+		ObjectAction systemObjectAction =
```

Comment:
Could you delete this variable `systemObjectAction` in the end of the test, please?

Reply:
You could delete `objectAction` too  :grimacing: